### PR TITLE
RPC camera execution

### PR DESCRIPTION
**Pull request by Somac**
https://jira.nls.fi/browse/OS-442 & https://jira.nls.fi/browse/OS-441

Cesium overrides mostly in this pull request

### CODE REVIEW

File: bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js [hunk 1]
```diff
@@ -454,6 +456,122 @@ class MapModuleOlCesium extends MapModuleOl {
...
+        const camera = options && options.heading && options.roll && options.pitch
+            ? { heading: options.heading,
+                roll: options.roll,
+                pitch: options.pitch } : undefined;
+
+        if (options && options.animation) {
+            this._flyTo(location[0], location[1], cameraHeight, animationDuration, camera);
+            if (suppressEnd !== true) {
+                setTimeout(() => {
+                    this.notifyMoveEnd();
```
Comment (ZakarFin) [inline bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js]: Please check if the AfterMapMoveEvent is triggered on 3D without this (as it was on 2D)?

File: bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js [hunk 2]
```diff
@@ -454,6 +456,122 @@ class MapModuleOlCesium extends MapModuleOl {
...
+        let flyToParams = { destination: Cesium.Cartesian3.fromDegrees(x, y, z) };
+        flyToParams = duration ? { ...flyToParams, duration: duration } : flyToParams;
+        flyToParams = cameraAngles ? { ...flyToParams, orientation: cameraAngles } : flyToParams;
+        flyToParams = complete ? { ...flyToParams, complete: complete } : flyToParams;
+        flyToParams = cancel ? { ...flyToParams, cancel: cancel } : flyToParams;
+        camera.flyTo(flyToParams);
+    }
+
+    adjustZoom (zoom) {
+        if (zoom === null || zoom === undefined) {
```
Comment (ZakarFin) [inline bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js]: Should this handle the zoomToExtent?

**Comment by ZakarFin** [comment]
Looks good for merging. Just check that the event is triggered and handle zoomToExtent if possible.

File: bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js (hunk 1 above)
Comment (Somac) [reply bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js]: It is triggered also, so removing this

File: bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js (hunk 2 above)
Comment (Somac) [reply bundles/paikkatietoikkuna/mapmodule/mapmodule.olcs.js]: I will handle it on centerMap for now, should probably finetune those values and/or logic on this.

**Pull request merged**